packages/react-examples/src/react/Facepile/Facepile.AddFace.Example.tsx

```diff
@@ -10,27 +10,24 @@ const overflowButtonProps = {
 
 export const FacepileAddFaceExample: React.FunctionComponent = () => {
   const [numberOfFaces, setNumberOfFaces] = React.useState(0);
-  const [addFaceAnnounced, setAddFaceAnnounced] = React.useState<JSX.Element | undefined>();
 
   const personas = React.useMemo(() => facepilePersonas.slice(0, numberOfFaces), [numberOfFaces]);
 
   const addButtonProps = React.useMemo(
     () => ({
       ariaLabel: 'Add a new person to the Facepile',
       onClick: (ev: React.MouseEvent<HTMLButtonElement>) => {
-        const currentFaces = numberOfFaces + 1;
-        setNumberOfFaces(currentFaces);
-        setAddFaceAnnounced(
-          <Announced message={`New person added, currently ${currentFaces} in the Facepile.`} aria-live="assertive" />,
-        );
+        setNumberOfFaces(numberOfFaces + 1);
       },
     }),
     [numberOfFaces],
   );
 
   return (
     <>
-      {addFaceAnnounced}
+      {numberOfFaces > 0 ? (
```

Comment:
Went with this approach for clarity. Let me know if `numberOfFaces || <Announced>` would be preferred.